### PR TITLE
[react-simple-maps] update `ZoomableGroupProps` interface

### PR DESCRIPTION
**Pull request by kopach**
- `onZoomStart` and `onZoomEnd` methods are not available anymore (here are docs with all available methods/props https://www.react-simple-maps.io/docs/zoomable-group/, here is code: https://github.com/zcreativelabs/react-simple-maps/blob/master/src/components/useZoomPan.js, here is some mentioning of that: https://github.com/zcreativelabs/react-simple-maps/issues/201)
- `onMoveStart`, `onMove`, `onMoveEnd` and `filterZoomEvent` methods definitions are updated.
  - code is here: https://github.com/zcreativelabs/react-simple-maps/blob/master/src/components/useZoomPan.js#L43,  https://github.com/zcreativelabs/react-simple-maps/blob/master/src/components/useZoomPan.js#L51, https://github.com/zcreativelabs/react-simple-maps/blob/master/src/components/useZoomPan.js#L62, https://github.com/zcreativelabs/react-simple-maps/blob/master/src/components/useZoomPan.js#L67)

### CODE REVIEW

**Comment by kopach** [comment]
Hi guys (@thepocp @pronebird @komenank). Would you mind, please, to check & merge this any soon? I  won't be able to work on this the next couple of weeks (if there would be such a need), so I would like to have this done within a couple of following days, if possible. Thanks.

File: types/react-simple-maps/react-simple-maps-tests.tsx [hunk 1]
```diff
@@ -8,7 +8,25 @@ const Map = () => (
...
     >
-        <ZoomableGroup center={[0, 20]} disablePanning={true} filterZoomEvent={(event: any) => event.type === 'wheel'}>
+        <ZoomableGroup
+            center={[0, 20]}
+            disablePanning={true}
+            filterZoomEvent={(event: any) => event.type === 'wheel'}
+            onMoveStart={({ coordinates, zoom }, event) => {
+                console.log('coordinates: ', coordinates);
+                console.log('zoom: ', zoom);
+                console.log('event: ', event);
```
Comment (peterblazejewicz) [inline types/react-simple-maps/react-simple-maps-tests.tsx:18]:
```suggestion
```
or replace with types assertions against those symbols, e.g.:
```ts
cordinates; // $ExpectType ...
```

File: types/react-simple-maps/react-simple-maps-tests.tsx [hunk 2]
```diff
@@ -8,7 +8,25 @@ const Map = () => (
...
+            }}
+            onMove={({ x, y, k, dragging }, event) => {
+                console.log('x, y, k, dragging: ', x, y, k, dragging);
+                console.log('event: ', event);
```
Comment (peterblazejewicz) [inline types/react-simple-maps/react-simple-maps-tests.tsx:22]:
```suggestion
```

File: types/react-simple-maps/react-simple-maps-tests.tsx [hunk 3]
```diff
@@ -8,7 +8,25 @@ const Map = () => (
...
+            }}
+            onMoveEnd={({ coordinates, zoom }, event) => {
+                console.log('coordinates: ', coordinates);
+                console.log('zoom: ', zoom);
+                console.log('event: ', event);
```
Comment (peterblazejewicz) [inline types/react-simple-maps/react-simple-maps-tests.tsx:27]:
```suggestion
```

File: types/react-simple-maps/tsconfig.json [hunk 4]
```diff
@@ -1,7 +1,10 @@
 {
     "compilerOptions": {
         "module": "commonjs",
-        "lib": ["es6"],
+        "lib": [
+            "es6",
+            "DOM"
```
Comment (peterblazejewicz) [inline types/react-simple-maps/tsconfig.json:6]:
```suggestion
            "es6"
```
was the dependency on `DOM` required only due to `console.log` usage?

File: types/react-simple-maps/index.d.ts [hunk 5]
```diff
@@ -7,6 +7,7 @@
 // TypeScript Version: 2.8
```
Comment (peterblazejewicz) [inline types/react-simple-maps/index.d.ts:7]:
```suggestion
```

File: types/react-simple-maps/index.d.ts [hunk 6]
```diff
@@ -7,6 +7,7 @@
...
 
 import { GeoPath, GeoProjection } from 'd3-geo';
+import { D3ZoomEvent } from 'd3-zoom';
```
Comment (peterblazejewicz) [inline types/react-simple-maps/index.d.ts:10]:
```ts
import { D3ZoomEvent } from 'd3-zoom';
```
v1 has dependency to v1 of d3-zoom. You should update this package `tsconfig.json`
to resolve 3 different version of d3-zoom properly:
```json
"paths": {
    "d3-zoom": [
        "d3-zoom/v1"
    ]
},
```

Comment (kopach) [reply types/react-simple-maps/index.d.ts:9]: done

Comment (kopach) [reply types/react-simple-maps/index.d.ts:9]: done

File: types/react-simple-maps/tsconfig.json (hunk 4 above)
Comment (kopach) [reply types/react-simple-maps/tsconfig.json]: Not sure already, most probably yes. Let's check if tests would pass without `DOM`

File: types/react-simple-maps/index.d.ts (hunk 5 above)
Comment (kopach) [reply types/react-simple-maps/index.d.ts]: done

File: types/react-simple-maps/tsconfig.json (hunk 4 above)
Comment (kopach) [reply types/react-simple-maps/tsconfig.json]: ok, just confirmed, it's not due to `console.log`. Tests fail without this. see results here https://github.com/DefinitelyTyped/DefinitelyTyped/runs/4152944086?check_suite_focus=true

Comment (peterblazejewicz) [reply types/react-simple-maps/tsconfig.json:6]: oh, yes, implicit dependnecy. Anyway, removing `console` would be nice, if you don't mind

File: types/react-simple-maps/react-simple-maps-tests.tsx (hunk 1 above)
Comment (kopach) [reply types/react-simple-maps/react-simple-maps-tests.tsx]: done

File: types/react-simple-maps/react-simple-maps-tests.tsx (hunk 2 above)
Comment (kopach) [reply types/react-simple-maps/react-simple-maps-tests.tsx]: done

File: types/react-simple-maps/react-simple-maps-tests.tsx (hunk 3 above)
Comment (kopach) [reply types/react-simple-maps/react-simple-maps-tests.tsx]: done

File: types/react-simple-maps/tsconfig.json (hunk 4 above)
Comment (kopach) [reply types/react-simple-maps/tsconfig.json:6]: yes, this is done already. `console.log` removed and replaced with `// $ExpectType` statements

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@kopach thx!

**Comment by kopach** [comment]
Ready to merge

**Pull request merged**